pom.xml

```diff
@@ -102,12 +102,12 @@
             <dependency>
                 <groupId>org.codehaus.plexus</groupId>
                 <artifactId>plexus-utils</artifactId>
-                <version>4.0.2</version>
+                <version>3.6.0</version>
             </dependency>
             <dependency>
                 <groupId>org.codehaus.plexus</groupId>
                 <artifactId>plexus-xml</artifactId>
-                <version>4.0.0</version>
+                <version>3.0.2</version>
```

Comment:
p-x is NOT needed with p-u 3.x (it is contained). Only with maven 4, where p-u 4.x is used p-x is needed as it was moved out from p-u.